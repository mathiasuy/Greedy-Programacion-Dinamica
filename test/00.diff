1,9c1,2
< 1>Cantidad: 1
< 2>Cantidad: 2
< 3>Cantidad: 1
< 4>Cantidad: 2
< 5>Volumen: 10
< 6>Volumen: 50
< 7>Volumen: 50
< 8>Volumen: 40
< 9>Fin.
---
> ==4364== error calling PR_SET_PTRACER, vgdb might block
> 1>
\ No newline at end of file
