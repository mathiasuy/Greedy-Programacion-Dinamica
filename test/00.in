max_cantidad_intervalos 
1
1-15-1
max_cantidad_intervalos 
2
1-15-1
15-30-1
max_cantidad_intervalos 
2
1-15-1
3-10-1
max_cantidad_intervalos 
3
1-15-1
3-20-1
15-30-1
max_volumen_intervalos
1
1-15-10
max_volumen_intervalos 
2
1-15-30
15-30-20
max_volumen_intervalos 
2
1-15-20
3-10-50
max_volumen_intervalos 
3
1-15-10
3-20-40
15-30-20
Fin
