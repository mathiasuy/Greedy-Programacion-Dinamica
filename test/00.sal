==4587== Use of uninitialised value of size 8
==4587==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4587==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
==4587== 
==4587== Invalid write of size 8
==4587==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4587==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
==4587==  Address 0x0 is not stack'd, malloc'd or (recently) free'd
==4587== 
==4587== 
==4587== Process terminating with default action of signal 11 (SIGSEGV)
==4587==  Access not within mapped region at address 0x0
==4587==    at 0x109D77: max_cantidad(intervalo_t const*, unsigned int) (in /mnt/c/p3/tareap3ant/principal)
==4587==    by 0x1094D8: main (in /mnt/c/p3/tareap3ant/principal)
==4587==  If you believe this happened as a result of a stack
==4587==  overflow in your program's main thread (unlikely but
==4587==  possible), you can try to increase the size of the
==4587==  main thread stack using the --main-stacksize= flag.
==4587==  The main thread stack size used in this run was 8388608.
1>timeout: the monitored command dumped core
Segmentation fault (core dumped)
