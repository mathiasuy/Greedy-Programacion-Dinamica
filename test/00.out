1>Cantidad: 1
2>Cantidad: 2
3>Cantidad: 1
4>Cantidad: 2
5>Volumen: 10
6>Volumen: 50
7>Volumen: 50
8>Volumen: 40
9>Fin.
